crates/templates/src/run.rs

```diff
@@ -405,6 +424,29 @@ impl Run {
             .build()
             .expect("can't fail due to no partials support")
     }
+
+    fn partials(&self) -> anyhow::Result<impl liquid::partials::PartialCompiler> {
+        let mut partials = Partials::empty();
+
+        if let Some(partials_dir) = self.template.partials_dir() {
+            for partial_file in std::fs::read_dir(partials_dir)? {
+                let partial_file = partial_file?;
+                if !partial_file.file_type().is_ok_and(|t| t.is_file()) {
+                    anyhow::bail!("Non-file in partials directory: {partial_file:?}");
+                }
+                let partial_name = partial_file
+                    .file_name()
+                    .into_string()
+                    .map_err(|f| anyhow!("Unusable partial name {f:?}"))?;
```

Comment:
```suggestion
                    .context("Unusable partial name")?;
```

Reply:
Could you elaborate on why `context` is better here?  The error case for `OsString::into_string()` is the unstringable file name, not a rich error, so I feel like `context` would lead to something like:

```
...
   1. loading template
   2: i'm mad as heck
   3: /bish/bash/bosh/zootlewurdle.liquid
   4: unusable partial name
```

which feels less obvious than "unusable partial name /bish/bash/bosh/zootlewurdle.liquid".

(Although the error case is hopefully largely moot, because WHO THE HECK SHIPS THINGS WITH UNSTRINGABLE FILE NAMES ANYWAY and NO LANN THIS IS NOT A CHALLENGE.)

Reply:
Ah sorry - I blindly assumed that the error was a meaningful error type and not just the filename itself. Nevermind!